packages/extension-sdk/templates/display/src/display.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+	<div>Value: {{ value }}</div>
+</template>
+
+<script>
+export default {
+	props: {
+		value: String,
+	},
+};
+</script>
```

Comment:
Can we default this to the composition API?

```vue
<script>
import { defineComponent } from 'vue';

export default defineComponent({
  etc
});
</script>

Reply:
As `defineComponent()` is essentially just a type hint, the Vue docs suggest to use `export default { ... }` when using Javascript in SFCs and `export default defineComponent({ ... })` when using Typescript in SFCs (see [this](https://v3.vuejs.org/guide/single-file-component.html#introduction) and [this](https://v3.vuejs.org/guide/typescript-support.html#defining-vue-components)).
I was planning to add it to the Typescript version of those templates. Do we want to have it in the Javascript templates as well?

Reply:
Nope!